core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/AiServicesRecorder.java

```diff
@@ -170,6 +171,29 @@ public T apply(SyntheticCreationalContext<T> creationalContext) {
                                 quarkusAiServices.retrievalAugmentor(supplier.get());
                             }
                         }
+                    } else if (info.getContentRetrieverSupplierClassName() != null) {
```

Comment:
I don't understand what we mean by "automatically wrap into a default" mentioned by @langchain4j

Reply:
What I meant was: if there is a `ContentRetriever` and no `RetrievalAugmentor`, `DefaultRetrievalAugmentor` should be created wrapping an existing `RetrievalAugmentor`:
```java
return DefaultRetrievalAugmentor.builder()
    .contentRetriever(retriever)
                .build();
```

Reply:
AFAIU Langchain4j wraps it automatically when you call `aiServices.contentRetriever()`: https://github.com/langchain4j/langchain4j/blob/0.27.1/langchain4j/src/main/java/dev/langchain4j/service/AiServices.java#L333